### PR TITLE
Makes ash heretic immune to fire

### PR DESCRIPTION
**Pull request by FriendlyContractor**
## About The Pull Request
Makes ash path heretic immune to being set on fire

## Why It's Good For The Game
People always bump into you in combat and end up seting you on fire as well which makes ash abilities allowing you to set others on fire a hinderance rather than uselfull ability. This tiny PR fixes it
## Changelog
:cl:
tweak: Ash heretics immune to being set on fire
/:cl:

### CODE REVIEW

File: code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm [hunk 1]
```diff
@@ -9,6 +9,10 @@
 	cost = 1
 	route = PATH_ASH
 
+/datum/eldritch_knowledge/base_ash/on_gain(mob/user)
+	. = ..()
+	ADD_TRAIT( user, TRAIT_NOFIRE,MAGIC_TRAIT)
```
Comment (Archanial) [inline code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm]: clear this on loss

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm (hunk 1 above)
Comment (FriendlyContractor) [reply code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm]: Fixed. Tested in game and works as intended

**Comment by zeskorion** [comment]
they shouldnt get fire immunity as a basic trait

**Comment by FriendlyContractor** [comment]
> 
> 
> they shouldnt get fire immunity as a basic trait

It fits with their fire themed lore nicely and most importantly as i mentioned when they get fiery blade upgrade which makes blade ignite people on hit those people inavoidably end up bumping into you and transfering portion of their flames to you which ultimately feels like this ability is a downgrade rather than something uselfull. We already have roundstart race which is immune to being set on fire and it has not been impacting the game negatively

**Comment by ItsTisme** [comment]
wont there clothing burn still? randomly dropping everything on you still sounds like a huge draw back

**Comment by FriendlyContractor** [comment]
> 
> 
> wont there clothing burn still? randomly dropping everything on you still sounds like a huge draw back

No - worn clothes cannot burn. I also tested this in game to make double sure and it checks out

**Comment by Howluinb** [comment]
Did you make it so that they would still need to ascend in order to be immune to high pressure? It would prevent grief, to some degree, as they would not be able to plasmaflood - only fight with fire.

**Comment by BrumalLake** [comment]
is this the trait that makes them instantly get extinguished cause that's a little conspicuous
and this should be on mark of ash since that's when they can actually start igniting people

**Comment by FriendlyContractor** [comment]
> 
> 
> Did you make it so that they would still need to ascend in order to be immune to high pressure? It would prevent grief, to some degree, as they would not be able to plasmaflood - only fight with fire.

The only thing it protects from is being set on fire. It doest not protect from pressure or heat

**Review by Archanial — APPROVED** [review]

**Pull request merged**